src/main/java/de/codingair/tradesystem/utils/database/migrations/sqlite/CreateTradeLogTableMigration.java

```diff
@@ -0,0 +1,24 @@
+package de.codingair.tradesystem.utils.database.migrations.sqlite;
+
+import de.codingair.tradesystem.utils.database.migrations.Migration;
+
+public class CreateTradeLogTableMigration implements Migration {
+    @Override
+    public String getStatement() {
+        return "CREATE TABLE IF NOT EXISTS tradelog (\n"
+                + "	id integer PRIMARY KEY,\n"
+                + "	player1 varchar NOT NULL,\n"
```

Comment:
don't know if you can add a varchar length. If so, pls use 16 characters.

Reply:
Done